README.md

```diff
@@ -181,6 +180,18 @@ generateOpenAPI({
 })
 ```
 
+#### Understanding mapFilePathToHTTPRoute
+
+```bash
+src
+└── pages
+    └── api
+        └── public
+            └── hello
+                └── index.ts
+```
+
+In the example above, the input file path is `/path/to/src/pages/api/public/hello/index.ts`. The mapFilePathToHTTPRoute function removes the string `src/pages/api/public`, the `.ts` extension, and the `/index` portion of the string, resulting in the output of `/hello`. The resulting string represents the HTTP route.
```

Comment:
This should be a docstring rather than in the readme I think, it's more of an internal detail than a feature of nextlove